crates/ra_hir_def/src/nameres.rs

```diff
@@ -115,6 +115,9 @@ pub struct ModuleData {
     pub definition: Option<FileId>,
 
     pub impls: Vec<ImplId>,
+
+    /// alias information (Name => Alias)
+    pub aliases: FxHashMap<Name, Name>,
```

Comment:
This info is only need during name-resolutrion process, not afterweards, so this field should be a part of `ModCollector`

Reply:
Yes, I agreed.